buildLinux.js

```diff
@@ -60,49 +46,4 @@ function spawnChecked(cmd, args, options) {
 
   return rv;
 }
-
-function syncRepo(dir, treeish) {
-  try {
-    spawnChecked("git", ["fetch", "--all"], { cwd: dir, stdio: "inherit" });
-  } catch (e) {
-    // Ignore errors due to being at a detached head.
-  }
-
-  spawnChecked("git", ["reset", "--hard", treeish], { cwd: dir, stdio: "inherit" });
-}
-
-function assert(value, msg) {
-  if (!value) {
-    throw new Error(msg);
-  }
-}
-
-function getChromiumDeps() {
-  const text = fs.readFileSync("DEPS", "utf8");
-  let results = {
-    v8: "",
-    skia: "",
-    webrtc: "",
-    boringssl: ""
-  };
-
-  let match = /'v8_revision': '(.*?)'/.exec(text);
-  assert(match, "Could not find V8 revision");
-  results.v8 = match[1];
-
-  match = /'skia_revision': '(.*?)'/.exec(text);
-  assert(match, "Could not find skia revision");
-  results.skia = match[1];
-
-  match = /'https:\/\/github.com\/replayio\/chromium-webrtc.git' \+ '@' \+ '(.*?)'/.exec(
-    text
-  );
-  assert(match, "Could not find webrtc revision");
-  results.webrtc = match[1];
-
-  match = /'boringssl_revision': '(.*?)'/.exec(text);
-  assert(match, "Could not find boringssl revision");
-  results.boringssl = match[1];
-
-  return results;
-}
+exports.spawnChecked = spawnChecked;
```

Comment:
Instead, let's move spawnChecked out of this file, since this is top-level.  "common.js" is fine :)